### PR TITLE
Remove unneeded Sort over Sort

### PR DESCRIPTION
**Pull request by akuzm**
We would add extra Sort nodes when adjusting the children of space
partitioning MergeAppend under ChunkAppend. This is not needed because
MergeAppend plans add the required Sort themselves, and in general no
adjustment seems to be required for the MergeAppend children
specifically there.

Disable-check: force-changelog-file

### CODE REVIEW

**Review by svenklemm — APPROVED** [review]
LGTM

File: tsl/test/sql/CMakeLists.txt [hunk 1]
```diff
@@ -134,6 +134,7 @@ set(SOLO_TESTS
     cagg_invalidation
     move
     reorder
+    transparent_decompression-${PG_VERSION_MAJOR}
```
Comment (svenklemm) [inline tsl/test/sql/CMakeLists.txt:137]: Why do we need this in SOLO_TESTS?

Comment (akuzm) [reply tsl/test/sql/CMakeLists.txt:137]: Just a debugging leftover, I'll remove it

**Review by nikkhils — APPROVED** [review]
LGTM!

**Pull request merged**